docs/pt-br/utilities.md

```diff
@@ -124,6 +124,22 @@ import { formatCEP } from '@brazilian-utils/brazilian-utils';
 formatCEP('92500000'); // 92500-000
 ```
 
+## isValidProcessoJuridico
+
```

Comment:
Missing some description.

Reply:
I think you forgot to add `en` docs too.

Reply:
True, I didn't notice the english docs, added the descriptions on both!